dm/ctl/master/config.go

```diff
@@ -81,6 +202,9 @@ func exportCfgsFunc(cmd *cobra.Command, args []string) error {
 		common.PrintLinesf("can not get directory")
 		return err
 	}
+	if dir == "" {
+		dir = "configs"
```

Comment:
where is dir set before this modification 🤔

Reply:
I want to combine this flag for `config export` and `config source` sub commands.